### PR TITLE
[ASM] - Expander - command argument bug fix

### PR DESCRIPTION
**Pull request by andrew-paloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
EXPANDR-5754

## Description
Update the enums used for alert statuses

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexXpanse/README.md [hunk 1]
```diff
@@ -7,22 +7,22 @@ Cortex Xpanse Active ASM is the best in class External Attack Surface Management
...
 
 ## Expanse - Alert Enrichment
 This playbook handles ASM alerts by enriching asset information via integrations with Cloud Service Providers and other IT and Security tools.
-![Xpanse - Alert Enrichment](https://raw.githubusercontent.com/demisto/content/5e6599c239629219147885acd54cc2cbde147034/Packs/CortexXpanse/doc_files/Xpanse_-_Alert_Enrichment.png)
+![Xpanse - Alert Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexXpanse/doc_files/Xpanse_-_Alert_Enrichment.png)
 
 ## Expanse - Alert Self-Enrichment
 This playbook handles ASM alerts by enriching alert information with Xpanse service and asset details.
-![Xpanse - Alert Self-Enrichment](https://raw.githubusercontent.com/demisto/content/5e6599c239629219147885acd54cc2cbde147034/Packs/CortexXpanse/doc_files/Xpanse_-_Alert_Self-Enrichment.png)
+![Xpanse - Alert Self-Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/Packs/CortexXpanse/doc_files/Xpanse_-_Alert_Self-Enrichment.png)
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/README.md:24]:
```suggestion
![Xpanse - Alert Self-Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexXpanse/doc_files/Xpanse_-_Alert_Self-Enrichment.png)
```
This should fix validation and pre-commit errors

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
This should fix validation and pre-commit errors

**Review by johnnywilkes — APPROVED** [review]
@andrew-paloalto , Looks good!

**Comment by johnnywilkes** [comment]
@RosenbergYehuda , this is approved by Xpanse team.  When @andrew-paloalto takes out of draft mode, please merge when possible

**Review by ben-expanse — APPROVED** [review]
Tag Changes LGTM! Thanks Andrew 😄

**Review by RosenbergYehuda — APPROVED** [review]
Looks good! @andrew-paloalto

**Pull request merged**